node/src/crypto/error.rs

```diff
@@ -33,7 +40,7 @@ pub enum Error {
 
     /// Error resulting when decoding a type from a base64 representation.
     #[error("decoding error: {0}")]
-    FromBase64(#[from] DecodeError),
+    FromBase64(#[from] base64::DecodeError),
```

Comment:
I suspect this variant is not used.

Reply:
I believe you're right, but let's not deal with that in this PR, since it also affects the crypto error in casper_types.